lib/src/DatePicker/components/Calendar.tsx

```diff
@@ -136,12 +138,20 @@ export class Calendar extends React.Component<CalendarProps, CalendarState> {
     this.props.onChange(utils.mergeDateAndTime(day, date), isFinish);
   };
 
-  public handleChangeMonth = (newMonth: MaterialUiPickersDate, slideDirection: SlideDirection) => {
+  public handleChangeMonth = async (
```

Comment:
I am not sure that we need async/await here for one option. It will lead to some additional bundle size. 
Use .then

Reply:
Fixed